webapp/channels/src/components/widgets/loading/loading_spinner.tsx

```diff
@@ -12,26 +12,22 @@ type Props = {
     style?: React.CSSProperties;
 }
 
-export default class LoadingSpinner extends React.PureComponent<Props> {
-    public static defaultProps: Props = {
-        text: null,
-    };
+const LoadingSpinner: React.FC<Props> = ({ text = null, style }) => {
+    return (
+        <span
+            id='loadingSpinner'
+            className={'LoadingSpinner' + (text ? ' with-text' : '')}
```

Comment:
Out of the scope, but do you mind using `classNames` for this? You should be able to do something like this:
```suggestion
            className={classNames('LoadingSpinner', {'with-text': Boolean(text)})}
```